objects/Resources.816d84.json

```diff
@@ -43,7 +43,7 @@
   "Tooltip": false,
   "Transform": {
     "posX": -59.793,
-    "posY": 1.61,
+    "posY": 1.55,
```

Comment:
I think these are a little higher than they used to be.  Bump them down a bit, or was the previous height causing issues?
![image](https://user-images.githubusercontent.com/5797523/202824702-d5ad4219-0e16-4a6c-b14f-628a7a125d3c.png)

Reply:
Previous height
![image](https://user-images.githubusercontent.com/5797523/202824942-309900a9-a5b3-4ba0-9e7b-ca65001ba751.png)

Reply:
previous height was causing flickering when viewed from directly above, but there is probably a middle ground